tests/conftest.py

```diff
@@ -1,9 +1,7 @@
 import os
 
-from dotenv import find_dotenv, load_dotenv
-
 import django
-
+from dotenv import find_dotenv, load_dotenv
```

Comment:
Did you edit this manually ? If so, you should not and use the `pre-commit` hooks `isort`. If you find something inconsistent (there is), you should edit the isort config (in `pyproject.toml`) and run the hook manually against the files (`pre-commit run --all-files isort`).